bin/GameIndex.yaml

```diff
@@ -9759,6 +9759,8 @@ SLES-51289:
   name: "Gunfighter 2 - Legend of Jesse James"
   region: "PAL-M5"
   compat: 5
+  gameFixes:
+  - GIFFIFOHack # Fixes flickering textures.
```

Comment:
tabbing.

Reply:
Oh only 2 spaces vs 4 spaces (which is a tab)
Surprised it still worked.